src/config.ts

```diff
@@ -6,8 +6,8 @@ export const config = () => {
     version: '0.1',
     /** Configuration related with session */
     session: {
-      /** TimeOut in Minutes default 24 hours */
-      sessionTimeOut: 24 * UNIT_TIME.HOUR * MILLISECONDS,
+      /** TimeOut in Minutes default 10 hours */
+      sessionTimeOut: 10 * UNIT_TIME.HOUR * MILLISECONDS,
```

Comment:
Comment. 
24 hours was on purpose as it is the default timeout of OpenShift Console.

Reply:
Hmm, I see. But that's a problem, as I commented in the JIRA ticket. We have different values for backend token expiry and frontend session timeout. They should match; or, better than that, the frontend timeout should be set automatically according to the token expiry date

Reply:
so should I revert to 24 hours, and maybe we'll try another solution via another JIRA?

Reply:
Personally I'm fine if we use the same JIRA to update the backend to match this value.
I will leave to @aljesusg to decide if the session timeout should come from a backend service, as he was the original author of this feature.

Reply:
Well, that's not that easy, the backend value is configurable by user, whereas the frontend value is not. That's why I say it would be better that the frontend reads the token expiry date.